package.json

```diff
@@ -102,6 +102,9 @@
     "@edsc/geo-utils": "^1.0.4",
     "@edsc/smart-handoffs": "^1.0.5",
     "@edsc/timeline": "^1.1.7",
+    "@radix-ui/colors": "3.0.0",
```

Comment:
Why was `colors` added? I am not seeing the radix colors used anywhere. Which is good. We should be using the colors available in the scss variables when possible.